.github/workflows/build-apps.yml

```diff
@@ -335,78 +351,44 @@ jobs:
                   "url": $linux_x64_url
                 }
               }
-            }' > last_download.json
-            cat last_download.json
+            }' > out/last_download.json
+            cat out/last_download.json
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: out-download-json
+          path: out/last_download.json
 
       - name: List artifacts
         run: "ls -R out"
 
       - name: Authenticate to Google Cloud
+        if: ${{ env.IS_NIGHTLY == 'true' }}
         uses: 'google-github-actions/auth@v2.1.7'
         with:
           credentials_json: '${{ secrets.GOOGLE_CLOUD_DL_SA }}'
 
       - name: Set up Google Cloud SDK
+        if: ${{ env.IS_NIGHTLY == 'true' }}
         uses: google-github-actions/setup-gcloud@v2.1.2
         with:
           project_id: ${{ env.GOOGLE_CLOUD_PROJECT_ID }}
 
-      - name: Upload release files to public bucket
-        uses: google-github-actions/upload-cloud-storage@v2.2.1
-        with:
-          path: out
-          glob: 'Zoo*'
-          parent: false
-          destination: ${{ env.BUCKET_DIR }}
-
-      - name: Upload update endpoint to public bucket
+      - name: Upload nightly files to public bucket
+        if: ${{ env.IS_NIGHTLY == 'true' }}
         uses: google-github-actions/upload-cloud-storage@v2.2.1
         with:
           path: out
-          glob: 'latest*'
+          glob: '*'
           parent: false
-          destination: ${{ env.BUCKET_DIR }}
-
-      - name: Upload download endpoint to public bucket
-        uses: google-github-actions/upload-cloud-storage@v2.2.1
-        with:
-          path: last_download.json
-          destination: ${{ env.BUCKET_DIR }}
+          destination: 'dl.kittycad.io/releases/modeling-app/nightly'
 
-      - name: Upload release files to Github
-        if: ${{ github.event_name == 'release' }}
+      - name: Create draft release
         uses: softprops/action-gh-release@v2
+        if: ${{ env.IS_RELEASE == 'true' }}
         with:
+          name: ${{ env.VERSION }}
+          tag_name: ${{ env.VERSION }}
+          draft: true
+          generate_release_notes: true
```

Comment:
[comment] This eliminates the need to copy the release notes out of the terminal? Sick!

Reply:
Yes! Same as checking the box when creating a release in the github UI